src/create_disk.sh

```diff
@@ -425,6 +425,8 @@ ostree config --repo $rootfs/ostree/repo set sysroot.bootloader "${bootloader_ba
 # Opt-in to https://github.com/ostreedev/ostree/pull/1767 AKA
 # https://github.com/ostreedev/ostree/issues/1265
 ostree config --repo $rootfs/ostree/repo set sysroot.readonly true
+ostree config --repo $rootfs/ostree/repo set sysroot.bls-append 'grub_users,""'
```

Comment:
I believe that this code needs to be conditional on architectures that use grub.  Or hmm, perhaps the simpler way to handle this is under `if test "$bootloader_backend" != none; then ...`

To say it another way, I suspect that the combined result of these PRs might cause `zipl` on s390x to fail with an "unknown option `grub_users`".

Reply:
Thanks for the explanation! I think I understand the issue better now. I agree that this could be more clean - I think adding the `if` statement you suggested would be a good way forward.